wordpress-mobile/WordPress-iOS
# No Results View: fix image hide/show on iPhone

Ref #9504 

When rotating on an iPhone, the image in the No Results View hides in landscape, and shows in portrait. The hide/show animation didn't work properly - the image appeared to kind of slide in/out. This change corrects that.

![broken](https://user-images.githubusercontent.com/1816888/45170105-5605f880-b1bc-11e8-9f16-5593b337ca34.gif)


**To test:**
- On an iPhone, go to a page that shows the No Results View (ex: My Sites > all sites hidden, Reader > Saved Posts, Medial Library > Giphy).
- Rotate the phone.
- Verify the image hiding/showing looks like:

![nrv_animation_fixed](https://user-images.githubusercontent.com/1816888/45169911-d546fc80-b1bb-11e8-87a4-a58749a9be5f.gif)
